python/tests/res/enkf/test_res_config.py

```diff
@@ -20,17 +20,50 @@
 
 from res.enkf import ResConfig, SiteConfig, AnalysisConfig
 
+config_defines = {
+        "<USER>"         : "TEST_USER",
+        "<SCRATCH>"      : "scratch/ert",
+        "<CASE_DIR>"     : "the_extensive_case",
+        "<ECLIPSE_NAME>" : "XYZ"
+        }
+
+config_data = {
+        "RUNPATH"          : "<SCRATCH>/<USER>/<CASE_DIR>/realization-%d/iter-%d",
+        "NUM_REALIZATIONS" : 10,
+        "MAX_RUNTIME"      : 23400,
+        "MIN_REALIZATIONS" : "50%",
+        "MAX_SUBMIT"       : 13
+        }
+
+def expand_config_data():
+    for define_key in config_defines:
+        for data_key in config_data:
+            if type(config_data[data_key]) is str:
```

Comment:
`if isinstance(config_data[data_key], str): ...` and could you add a short description of what (and why) `expand_config_data` does?